index.html

```diff
@@ -11,40 +11,251 @@
 </head>
 
 <body>
-  <nav class="navbar">
-    <h2>My Logo</h2>
-    <img class="icon" src="images/Union.jpg" alt="Union">
-  </nav>
-  <section class="section">
-    <header class="title">
-      <h1>I’m Ilham
-        Glad to see you!</h1>
-    </header>
-    <header class="para">
-      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
-        work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact me.</p>
-    </header>
-    <h3 class="h3">LET’S CONNECT</h3>
-    <div class="icons">
-      <ul class="headline-img">
+  <header class="navbar">
+    <nav class="container">
+      <a class="logo" href="">My Logo</a>
+      <img src="images/Union.jpg" alt="Union">
+    </nav>
+  </header>
+  <section class="hero">
+    <div class="container">
+      <h2 class="primary-text">I’m Ilham <br> Glade to see you!</h2>
+    </div>
+    <p class="secondary-text">
+      I’m a software developer! I can help you build a product ,
+      feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded,
+      don’t hestiate to contact me.
+    </p>
+
+    <h4 class="headline">LET’S CONNECT</h4>
+    <ul class="headline-img">
+      <li><a href="#"><img src="images/Vector(3).svg" alt="twitter-icon" /></a></li>
+      <li><a href="#"><img src="images/Linkedinicon.svg" alt="linkdn-icon" /></a></li>
+      <li><a href="#"><img src="images/Vector(2).svg" alt="monster-icon" /></a></li>
+      <li><a href="#"><img src="images/Vector(1).svg" alt="gitbubicon" /></a></li>
+      <li><a href="#"><img src="images/Vector.svg" alt="thumb" /></a></li>
+    </ul>
+  </section>
+
+  <section>
+    <div class="cards-container">
+      <ul class="cards-list flex-center-culomn">
+        <li class="card-item flex-center-culomn">
+          <img class="card-image" src="images/SnapshootPortfolio.jpg" alt="Portfolio Portfolio">
+          <div class="list-conatiner">
```

Comment:
- [ ] As already stated by the previous reviewer, please ensure there is enough spacing between the headline section and the works section

### Your Design 
![image](https://user-images.githubusercontent.com/43995830/159684213-e07d5761-a3b9-4f27-9a6d-1e28c1556799.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/159684743-25bcf62d-7119-4e4f-b30e-d301fa876fe8.png)